### PR TITLE
Display the members list for a committee

### PR DESCRIPTION
**Pull request by romain-pierre**
Closes [EM-60](http://jira.sensiolabs.net/browse/EM-60).

### CODE REVIEW

File: app/Resources/views/committee/members.html.twig [hunk 1]
```diff
@@ -6,6 +6,44 @@
 
 {% block banner '' %}
 
+{% block javascripts %}
+    <script type="application/javascript">
+        var allCheckbox = dom('#members-check-all');
```
Comment (hhamon) [inline app/Resources/views/committee/members.html.twig]: `allCheckboxesContainer` ?

File: app/Resources/views/committee/members.html.twig [hunk 2]
```diff
@@ -6,6 +6,44 @@
...
+        var membersCheckbox = dom("[name='members[]']");
```
Comment (hhamon) [inline app/Resources/views/committee/members.html.twig]: `membersCheckboxes` ?

File: app/Resources/views/committee/members.html.twig (hunk 1 above)
Comment (romain-pierre) [inline app/Resources/views/committee/members.html.twig]: Why container? This is an unique field.

Comment (hhamon) [inline app/Resources/views/committee/members.html.twig]: Nevermind... It targets the checkbox in the header section. I thought it meant "all checkboxes" elements.

**Pull request merged**